src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,31 @@
-export const TodoInfo = () => {};
+import { FC } from 'react';
+import { Todo } from '../../api/todos';
+import { User } from '../../api/users';
+import { UserInfo } from '../UserInfo';
+import usersFromServer from '../../api/users';
+
+interface Props {
+  todo: Todo;
+  // user?: User;
+}
+
+export const TodoInfo: FC<Props> = props => {
+  const { todo} = props;
+  const { id, completed, title, userId } = todo;
+
+  const filteredUser = usersFromServer.find(
+    (user: User): boolean => user.id === userId,
+  );
+
+  return (
+    <article
+      data-id={id}
+      className={completed ? 'TodoInfo TodoInfo--completed' : 'TodoInfo'}
```

Comment:
To set classes conditionally use [classnames](https://www.npmjs.com/package/classnames) or [clsx](https://www.npmjs.com/package/clsx) library